make test failing for all tests

I built nvtt using the master branch, using the defaults and all tests are currently failing on x86_64.

```bash
Running tests...
Test project /project/software/library/nvidia/nvidia-texture-tools/build-release
    Start 1: NVTT.TestSuite.Kodak.cuda
1/6 Test #1: NVTT.TestSuite.Kodak.cuda ........***Failed    0.03 sec
    Start 2: NVTT.TestSuite.Waterloo.cuda
2/6 Test #2: NVTT.TestSuite.Waterloo.cuda .....***Failed    0.02 sec
    Start 3: NVTT.TestSuite.Epic.cuda
3/6 Test #3: NVTT.TestSuite.Epic.cuda .........***Failed    0.02 sec
    Start 4: NVTT.TestSuite.Kodak.nocuda
4/6 Test #4: NVTT.TestSuite.Kodak.nocuda ......***Failed    0.02 sec
    Start 5: NVTT.TestSuite.Waterloo.nocuda
5/6 Test #5: NVTT.TestSuite.Waterloo.nocuda ...***Failed    0.02 sec
    Start 6: NVTT.TestSuite.Epic.nocuda
6/6 Test #6: NVTT.TestSuite.Epic.nocuda .......***Failed    0.02 sec

0% tests passed, 6 tests failed out of 6

Total Test time (real) =   0.16 sec

The following tests FAILED:
	  1 - NVTT.TestSuite.Kodak.cuda (Failed)
	  2 - NVTT.TestSuite.Waterloo.cuda (Failed)
	  3 - NVTT.TestSuite.Epic.cuda (Failed)
	  4 - NVTT.TestSuite.Kodak.nocuda (Failed)
	  5 - NVTT.TestSuite.Waterloo.nocuda (Failed)
	  6 - NVTT.TestSuite.Epic.nocuda (Failed)
Errors while running CTest
Makefile:119: recipe for target 'test' failed
make[1]: *** [test] Error 8
Makefile:8: recipe for target 'test' failed
make: *** [test] Error 2
```

--- a/src/nvtt/tests/testsuite.cpp
+++ b/src/nvtt/tests/testsuite.cpp
@@ -509,6 +509,9 @@ int main(int argc, char *argv[])
     nvtt::Context context;
     context.enableCudaAcceleration(!nocuda);
 
+    if (basePath.length() > 0) {
+        basePath.appendSeparator();
+    }
     basePath.append(set.basePath);
 
     FileSystem::changeDirectory(basePath.str());

